user_docs/en/userGuide.t2t

```diff
@@ -1497,7 +1497,7 @@ Toggles the announcement of new content in particular objects such as terminals
 ==== Play a sound when auto-suggestions appear ====[ObjectPresentationSuggestionSounds]
 Toggles announcement of appearance of auto-suggestions, and if enabled, NVDA will play a sound to indicate this.
 Auto-suggestions are lists of suggested entries based on text entered into certain edit fields and documents.
-For example, when you enter text into the search box in Start menu in Windows Vista and later, Windows displays a list of suggestions based on what you typed.
+For example, when you enter text into the search box in Start menu in Windows 8 and later, Windows displays a list of suggestions based on what you typed.

```

Comment:
Why is this changed? Have you tested that this? There is a search box in the start menu of vista, and windows 7 which gives suggestions, do these not work with NVDA?

Reply:
Ok I was prety sure that Windows introduced autocomplete feature to start menu in Windows 8 and I thought in Windows 7 and Vista this feature was present only in Windows explorer and the run box. Testing shows that suggestions are also shown in Windows 7. I don't know how it is in Windows Vista though. But I changed it back to Windows Vista. I think most users will not be confused by this.

Reply:
The suggestions are  also available on Windows  Vista. Having said that in my opinion it should be reworded. For a beginnner user reading this the immediate conclusion would be that if he/she is using Windows Vista or later when typing in the start menu the suggestion sound would play when suggestion appears. It is not the case ad least not on anything lover than Windows 10. To avoid this confusion it might make sense either to not use an example here, or note that it works only in Windows 10.